diff --git a/src/wasm/CMakeLists.txt b/src/wasm/CMakeLists.txt
index 20fc2e9fb..f411d9e63 100644
--- a/src/wasm/CMakeLists.txt
+++ b/src/wasm/CMakeLists.txt
@@ -18,4 +18,6 @@ set(wasm_SOURCES
 if (NOT MSVC)
   set_source_files_properties(wasm-debug.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
 endif()
+# needed for gcc to compile the llvm sources included here
+add_definitions(-D__STDC_FORMAT_MACROS)
 add_library(wasm OBJECT ${wasm_SOURCES})
+target_compile_features(wasm PRIVATE cxx_std_17)
